DQM/SiPixelPhase1Heterogeneous/plugins/SiPixelPhase1TrackComparisonHarvester.cc

```diff
@@ -0,0 +1,59 @@
+#include "DQMServices/Core/interface/DQMEDHarvester.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+class SiPixelPhase1TrackComparisonHarvester : public DQMEDHarvester {
+public:
+  explicit SiPixelPhase1TrackComparisonHarvester(const edm::ParameterSet&);
+  ~SiPixelPhase1TrackComparisonHarvester() override = default;
+  void dqmEndJob(DQMStore::IBooker& ibooker, DQMStore::IGetter& igetter) override;
+
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+private:
+  // ----------member data ---------------------------
+  const std::string topFolder_;
+};
+
+SiPixelPhase1TrackComparisonHarvester::SiPixelPhase1TrackComparisonHarvester(const edm::ParameterSet& iConfig)
+    : topFolder_(iConfig.getParameter<std::string>("topFolderName")) {}
+
+void SiPixelPhase1TrackComparisonHarvester::dqmEndJob(DQMStore::IBooker& ibooker, DQMStore::IGetter& igetter) {
+  MonitorElement* hpt_eta_tkAllCPU = igetter.get(topFolder_ + "/ptetatrkAllCPU");
+  MonitorElement* hpt_eta_tkAllCPUmatched_ = igetter.get(topFolder_ + "/ptetatrkAllCPUmatched");
+  MonitorElement* hphi_z_tkAllCPU_ = igetter.get(topFolder_ + "/phiztrkAllCPU");
+  MonitorElement* hphi_z_tkAllCPUmatched_ = igetter.get(topFolder_ + "/phiztrkAllCPUmatched");
```

Comment:
please do not use a trailing underscore `_` for local variables, only for data members:
```suggestion
  MonitorElement* hpt_eta_tkAllCPUmatched = igetter.get(topFolder_ + "/ptetatrkAllCPUmatched");
  MonitorElement* hphi_z_tkAllCPU = igetter.get(topFolder_ + "/phiztrkAllCPU");
  MonitorElement* hphi_z_tkAllCPUmatched = igetter.get(topFolder_ + "/phiztrkAllCPUmatched");
```

Reply:
done in [4d1eb85](https://github.com/cms-sw/cmssw/pull/37619/commits/4d1eb85cd174a8fec462964a77a845a935c54b82)